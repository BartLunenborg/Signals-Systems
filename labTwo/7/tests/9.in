0.8
10: [4,8,6,7,2,9,0,7,1,1]
200: [2,4,0,2,8,9,5,4,2,8,1,2,3,6,2,9,6,4,0,9,1,9,7,0,1,9,9,9,5,2,7,9,6,9,1,6,9,7,1,1,7,4,3,2,2,8,1,8,2,2,7,5,1,7,8,4,8,7,5,5,9,2,4,8,4,7,4,5,6,7,6,3,3,1,5,6,1,9,6,6,3,6,3,6,3,1,1,3,0,6,8,2,1,2,0,7,9,6,2,8,4,8,1,9,1,9,7,5,0,4,1,3,2,4,9,7,8,2,0,8,9,8,0,2,2,0,9,3,9,1,1,5,1,5,4,4,4,2,9,4,8,2,9,0,9,8,7,7,1,9,5,2,9,8,6,1,0,7,6,9,8,8,4,1,3,9,7,9,3,9,5,1,1,4,3,2,4,0,9,5,1,7,9,0,5,5,1,5,2,7,5,2,7,9,5,2,0,3,1,3]
