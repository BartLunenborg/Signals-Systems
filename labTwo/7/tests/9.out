76 0.80381
